geoproxy/main.go

```diff
@@ -172,16 +265,30 @@ func enableCors(w *http.ResponseWriter) {
 
 // Given a request send it to the appropriate url
 func handleRequestAndRedirect(res http.ResponseWriter, req *http.Request) {
-
-	fmt.Printf("Request received\n")
-	requestPayload := parseRequestBody(req)
-	url := getProxyUrl(requestPayload.Lat, requestPayload.Lng)
-
-	fmt.Printf("Conditional url attained\n")
-	logRequestPayload(requestPayload, url)
-
-	fmt.Printf("Request served to reverse proxy\n")
-	serveReverseProxy(url, res, req)
+	
+	if strings.Contains(req.URL.Path, "view") {
+		// View request
+		fmt.Printf("View request received\n")
+		requestPayload := parseViewRequestBody(req)
+		urls := getViewProxyUrl(requestPayload.LatLng1, requestPayload.LatLng2)
+		fmt.Printf("Conditional url(s) attained\n")
+		for url := range urls {
```

Comment:
maybe good to note that we're currently expecting urls to be of length 1 until we add support for routing to multiple servers